proposals/templates/proposals/proposal_pdf.html

```diff
@@ -415,18 +415,10 @@ <h2>{% trans "Informed consent formulieren" %}</h2>
                 <th>{% get_verbose_field_name "studies" "documents" "briefing" %}</th><td><a href="{{ BASE_URL }}{{ study_documents.briefing.url }}">{% trans "Download" %}</a></td>
             </tr>
             {% endif %}
-            <tr>
-                <th>{% get_verbose_field_name "studies" "study" "passive_consent" %}</th><td>{{ study.passive_consent|yesno:_("ja,nee") }}</td>
-            </tr>
-            {% if study.passive_consent %}
-                <tr>
-                    <th>{% get_verbose_field_name "studies" "study" "passive_consent_details" %}</th><td>{{ study.passive_consent_details }}</td>
-                </tr>
```

Comment:
Ideally we keep this in the PDF for older studies.

Moving the passive consent field below the if-statement would make it disappear if it's not used in the study, thus at least keeping the fields in the PDF's that do use passive consent.

Reply:
This is now fixed with a little extra logic!

Passive consent will show up in PDF's and diffs it has any value set other than `None`. So that will include older proposals with a decisive No answer. The school forms will now always show up in the PDF if present, regardless of the newer logic for `Study.needs_additional_external_forms()`. I think that catches everything.